freelance-agreement.md

```diff
@@ -0,0 +1,135 @@
+#dwyl Freelance Agreement
+
+![seedling](http://i.imgur.com/u6xFrZX.jpg)
+
+You are hereby _invited_ to _join us_ as a _**freelance contributor**_ of _**dwyl**_  - the app that helps you _**do what you love**_!
+
+This agreement has been crafted to **_clarify_ several questions** we have been asked by potential contributors to the project(s).
+_**In short**: What you contribute is **your own work**. You can still **keep using it for anything** you like and once it's part of dwyl's repositories, **so can we**. Remember that all our code is **open source so other people can use it too!**_.
+
+
+## Why have this agreement?
+
+This agreement is for _your_ peace of mind as a contributor (we want to make sure you have no doubts about what will happen to your contribution) as well as for the protection of **dwyl Ltd** and others using _**the codebase**_.
+
+We **_request_ that you _either_ physically or electronically _sign_ (a _printed or digital_ version respectively of) this agreement** and return it to Inês @**dwyl** _**before committing any code**_ to the project.
+
+## What are we agreeing to?
+
+As an individual contributor, you are contributing **_original works_** of code
+to the open source codebase ("**the codebase**") which forms the "**dwyl app**",
+maintained **dwyl ltd** and hosted on GitHub at: https://github.com/dwyl
+
+As a **contributor**, you hereby confirm that:
+
+1. The contributions you make to the codebase are **_your original work_**
+(_never copied from somewhere else_)
+2. By committing your code into a dwyl-owned repository, **you grant worldwide, and irrevocable but _non-exclusive_ copyright with no further consideration**
+3. This is an open source project, so in addition to **dwyl ltd**. using your (and all of our) contributions, the codebase may also be used by other people (3rd parties) without further consent or consideration.
++ Your contribution **does not grant you a _claim_ to any earnings** (past, present or future) of **dwyl Ltd**. or any other individuals or companies that use your code.
++ You **do not have any _obligation_ to provide support for your code**, however you are encouraged to remain a contributor/supporter for as long you find it personally beneficial/rewarding;
++ If you use the original works you contribute to the dwyl codebase in future projects we ask that you provide **attribution** back to the project where you originally made the contribution.
+
+We (_dwyl Ltd_.) agree to the following:
```

Comment:
I would favour using 'dwyl' and 'dwyl ltd.' interchangeably (defining this in the terms/definitions) as 'dwyl ltd' is very harsh and reduces readability. It's a long document and I would prefer it to 'flow'. @nelsonic Do you have strong objections to this?